app/views/verify/activated.html.slim

```diff
@@ -3,4 +3,4 @@
 h1.h3.my0 = t('idv.titles.activated')
 p.mt-tiny.mb0 = t('idv.messages.activated_html',
                   link: link_to(t('idv.messages.activated_link'), contact_path))
-= link_to 'Back', idv_url
+= link_to t('forms.buttons.back'), verify_path
```

Comment:
looks like @monfresh is getting rid of this exact same link in https://github.com/18F/identity-idp/pull/894/files ... there will be a merge conflict